src/include/cursor.h

```diff
@@ -57,7 +57,7 @@ struct __wt_cursor_backup {
     uint64_t bit_offset;  /* Current offset */
     uint64_t granularity; /* Length, transfer size */
 
-/* AUTOMATIC FLAG VALUE GENERATION START */
+    /* AUTOMATIC FLAG VALUE GENERATION START */
```

Comment:
This whitespace change is unexpected. Can you investigate it? Normally the opening comment for automatic flags is aligned at the left margin. This is also not part of the original change.

Reply:
Hi Sue, thanks for pointing this out. I believe it may have been changed by the editor when I was resolving a merge conflict but it was definitely not intended on my part, I will fix this immediately.